### PR TITLE
feature/cast-nRLC-to-RLC

### PR DESCRIPTION
**Pull request by Le-Caignec**
- [x]  Add one unit test (more in an other PR)
- [x]  cast nRLC to RLC (as POCO subgraph)
- [x]  upgrade old dep

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -51,3 +51,8 @@ export function getEventId(event: ethereum.Event): string {
     event.transaction.hash.toHex() + "_" + event.transactionLogIndex.toString()
   );
 }
+
+export function nRLCToRLC(value: BigInt): BigInt {
+  let divisor = BigInt.fromI32(1000000000);
+  return value.div(divisor);
+}
```
Comment (gfournieriExec) [inline src/utils.ts:58]: Maybe you you can re-use this function name for consistency 
https://github.com/iExecBlockchainComputing/PoCo-subgraph/blob/619bfa67720509cb0f3c9d7b3eeff9496681c8ed/src/utils.ts#L175

**Review by gfournieriExec — APPROVED** [review]

File: tests/unit/VoucherCreated.test.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    handleVoucherCreated(event);
+
+    assert.entityCount("Voucher", 1);
+    assert.entityCount("VoucherCreation", 1);
+
+    assert.fieldEquals("Voucher", voucherAddress, "voucherType", voucherTypeId);
+    assert.fieldEquals("Voucher", voucherAddress, "owner", owner);
+    assert.fieldEquals("Voucher", voucherAddress, "value", "2");
+    assert.fieldEquals("Voucher", voucherAddress, "balance", "2");
```
Comment (jeremyjams) [inline tests/unit/VoucherCreated.test.ts:60]: Very minor, maybe 
```suggestion
    assert.fieldEquals("Voucher", voucherAddress, "value", rlcValue);
    assert.fieldEquals("Voucher", voucherAddress, "balance", rlcValue);
```
with something like `let nRlcValue = BigInt.fromI32(rlcValue * 1_000_000_000); // in nRLC`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+  - datasetPrice
+  - workerpoolPrice
+  - sponsoredAmount
+  - drainedAmount
+  - refundedAmount
+  - topUpValue
+  - value
+
+- Add unit tests for Event:
+  - voucherCreated
```
Comment (jeremyjams) [inline CHANGELOG.md:16]: You can eventually add PR number like that https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/blob/main/CHANGELOG.md?plain=1#L9

**Review by jeremyjams — APPROVED** [review]
Great, thanks!

**Pull request merged**